osu.Game/Updater/SimpleUpdateManager.cs

```diff
@@ -30,9 +32,11 @@ private void load(OsuGameBase game)
             version = game.Version;
 
             if (game.IsDeployedBuild)
-                Schedule(() => Task.Run(checkForUpdateAsync));
+                CheckForUpdate();
         }
 
+        public override void CheckForUpdate() => Schedule(() => Task.Run(checkForUpdateAsync));
```

Comment:
public method that bypasses the `IsDeployedBuild` check? should probably be protected at most.